mysite/checkers/views.py

```diff
@@ -89,3 +93,15 @@ class game(View):
     def get(self, request):
         if request.method == 'GET':
             return HttpResponse(render(request,'checkers/game.html'))
+			
```

Comment:
new views should be in there own class

Reply:
Room is just part of game like create session.